staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -574,6 +574,8 @@ type VirtualMachineInstanceNetworkInterface struct {
 	InfoSource string `json:"infoSource,omitempty"`
 	// Specifies how many queues are allocated by MultiQueue
 	QueueCount int32 `json:"queueCount,omitempty"`
+	// PodConfigDone specifies if the corresponding pod interface is properly configured by CNI
+	PodConfigDone bool `json:"podConfigDone,omitempty"`
```

Comment:
How did this new parameter got in at this stage? Under the `virt-controller, multus: mutate cncf.io/networks pod annotations ` commit.

Reply:
You asked me to add this only when the parameter was added. I complied.